<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantItemView</class>
 <widget class="QWidget" name="ParticipantItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,4,1,1">
   <item>
    <widget class="QLabel" name="labelDisplayName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBarVolume">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="format">
      <string>%p</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAudio">
     <property name="text">
      <string>audio</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonVideo">
     <property name="text">
      <string>video</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
